Lib/test/test_grammar.py

```diff
@@ -444,6 +444,20 @@ def tellme(file=None):
         check_syntax_error(self, 'print ,')
         check_syntax_error(self, 'print >> x,')
 
+        with check_py3k_warnings(("Missing parentheses in call to 'print' is not supported", SyntaxWarning)):
```

Comment:
Just to check, does this check that **each** of the cases inside the `with` body generates the warning?

Reply:
Yes, it does